core/src/main/java/com/sequenceiq/cloudbreak/service/image/ImageCatalogService.java

```diff
@@ -691,6 +692,54 @@ private String createCRN(String accountId) {
                 .toString();
     }
 
+    @Override
+    public String getResourceCrnByResourceName(String resourceName) {
+        if (CDP_DEFAULT_CATALOG_NAME.equals(resourceName)) {
+            return CDP_DEFAULT_CATALOG_CRN;
+        } else if (legacyCatalogEnabled && CLOUDBREAK_DEFAULT_CATALOG_NAME.equals(resourceName)) {
+            return CLOUDBREAK_DEFAULT_CATALOG_CRN;
+        } else {
+            return imageCatalogRepository.findResourceCrnByNameAndWorkspaceId(resourceName, workspaceService.getForCurrentUser().getId())
+                    .orElseThrow(() -> NotFoundException.notFoundException("Image catalog", resourceName));
```

Comment:
I think we should talk about these default catalogs, I am not sure I get the logic here